main.go

```diff
@@ -59,6 +60,7 @@ const (
 	gatewayDebugEnvironmentVariable          = "GATEWAY_DEBUG"
 	gatewayVerboseEnvironmentVariable        = "VERBOSE"
 	logSecretsEnvironmentVariable            = "LOG_SECRETS"
+	targetSchemeAndAuthorityRewriteMap       = "TARGET_REWRITES"
```

Comment:
nit: Consistency with the above

```suggestion
	targetRewritesVariable       = "TARGET_REWRITES"
```